### DISCUSSION TITLE
2006/11/26/%E9%A1%BA%E5%BA%8F%E7%BB%93%E6%9E%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#133] [Announcements]
# 2006/11/26/%E9%A1%BA%E5%BA%8F%E7%BB%93%E6%9E%84

一. 程序执行顺序

https://blog.qizhen.xyz/2006/11/26/%E9%A1%BA%E5%BA%8F%E7%BB%93%E6%9E%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘伟印
2008年08月15日 13:37 编辑
图4好像是事件结构吧？

回复
